### ISSUE TITLE
Plex Not Syncing

### ISSUE THREAD

**Issue by McGriddle** [tidusjar/Ombi#3083]
**Describe the bug**
Plex no longer syncs so requests never get updated and notifications are never sent.

**To Reproduce**
1. Use ombi to add shows and movies.
2. Every few months that error starts showing up to where no requests get updated.

**Expected behavior**
Ombi syncs with plex, updates the state of a request, and sends the enabled notifications.

**Screenshots**
none

**Logs (Logs directory where Ombi is located)**
```
ombi         | info: Ombi.Schedule.Jobs.Plex.PlexContentSync[0]
ombi         |       Starting EP Cacher
ombi         | fail: Ombi.Schedule.Jobs.Plex.PlexEpisodeSync[2000]
ombi         |       Caching Episodes Failed
ombi         | System.OperationCanceledException: The operation was canceled.
ombi         |    at System.Net.Http.HttpClient.HandleFinishSendAsyncError(Exception e, CancellationTokenSource cts)
ombi         |    at System.Net.Http.HttpClient.FinishSendAsyncBuffered(Task`1 sendTask, HttpRequestMessage request, CancellationTokenSource cts, Boolean disposeCts)
ombi         |    at Ombi.Api.OmbiHttpClient.SendAsync(HttpRequestMessage request) in C:\projects\requestplex\src\Ombi.Api\OmbiHttpClient.cs:line 61
ombi         |    at Ombi.Api.Api.Request[T](Request request) in C:\projects\requestplex\src\Ombi.Api\Api.cs:line 38
ombi         |    at Ombi.Api.Plex.PlexApi.GetLibrarySections(String authToken, String plexFullHost) in C:\projects\requestplex\src\Ombi.Api.Plex\PlexApi.cs:line 120
ombi         |    at Ombi.Schedule.Jobs.Plex.PlexEpisodeSync.Cache(PlexServers settings) in C:\projects\requestplex\src\Ombi.Schedule\Jobs\Plex\PlexEpisodeSync.cs:line 71
ombi         |    at Ombi.Schedule.Jobs.Plex.PlexEpisodeSync.Execute(IJobExecutionContext job) in C:\projects\requestplex\src\Ombi.Schedule\Jobs\Plex\PlexEpisodeSync.cs:line 47
```

**Desktop (please complete the following information):**
 - OS: n/a. Server related.

**Ombi Version (please complete the following information):**
 - Version 3.0.4680 (but also failing on the previous version)
- Media Server: plex 1.16.1.1291 

Wiping the db and starting from scratch usually fixes the issue. Are there particular tables I can wipe for plex, sonarr, and radarr, so I don't have to setup Ombi all over again? I do have a lot of custom settings.

**Comment by ytsejam1138** [comment]
I'm having a similar issue.  I'm running Ombi v3.0.4680 in a Docker image on my Synology DS918+. It was working fine for a few days and then I started having sync issues with my Plex server. I ended up deleting everything and starting from scratch. Ombi finds my Plex server and can connect, it just can't sync any movies or tv shows. Importing users worked fine.  Plex Server (1.17.0.1709) is running on the Synology DS918+, but not in a Docker container.

**Logs**

[log-20190921.txt](https://github.com/tidusjar/Ombi/files/3638998/log-20190921.txt)

**Comment by neoKushan** [comment]
I found this issue while googling for the same problem, dismayed to see that it has been ignored and closed thus far.

However, I managed to figure out a solution/workaround that worked for me.  I have the same setup as @ytsejam1138 so I suspect this is a quirk of running Ombi in a container that's running on a Synology NAS, while Plex is installed as a Synology Package on the same machine.

In the Media server settings for plex, typing in your username and password then clicking "Load Servers" will give you some incorrect details. However it does at least pull the correct plex token. 

In the hostname, for me it was picking out my local IP and a completely wrong port (not even 32400).

I changed the hostname to be my _external_ hostname (The one my users and plex.tv can access).

For the port I selected 443.

I SSL.

I have my plex instance behind a reverse proxy so it can be accessed via https://my.server.url (with no port specified).
